src/Utility.java

```diff
@@ -15,9 +15,29 @@ protected static void displayMenu() {
         
     } */
 
-    /* protected ... ... handleUserMenuChoice(...) {
-        ...
-    } */
+    protected static void handleUserMenuChoice() {
+        Scanner input = new Scanner(System.in); 
+        System.out.println("Please enter selection: ");
+        String choice = input.nextLine(); 
+        if (choice == "1")
```

Comment:
Since you are reading the input as a String, you should use the equals method for choice to compare. 
`choice.equals("1"))`